x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -98,7 +98,7 @@ export interface UpdateProperties {
   timelineTitle: TimelineTitleOrUndefined;
   meta: MetaOrUndefined;
   machineLearningJobId: MachineLearningJobIdOrUndefined;
-  filters: PartialFilter[];
+  filters: PartialFilter[] | undefined;
```

Comment:
Is this change intentional?

Reply:
Yes, it's intentional. 
`UpdateProperties` is used for calculateVersion, which checks supplied properties in patchRules.
If any of parameter properties not undefined, it will increment version. 

As I made `PatchRulesOptions` optional, I need to also be able to pass filters as undefined. If it's not present, version will not be incremented